test/release/examples/primers/modules.chpl

```diff
@@ -0,0 +1,386 @@
+/*
+ * Module Primer
+ *
+ * This primer introduces the concept of modules, a concept for encapsulating
+ * code for use by other code.  It covers:
+ *   how to define a module
+ *   namespace control within a module
+ *   external access to a module's symbols
+ *   namespace control when using a module, including:
+ *     unlimited
+ *     explicit exclusion of symbols
+ *     explicit inclusion of symbols
+ *     renaming included symbols
+ *   cross-file module access
+ */
+
+
+/* A module is a grouping of code - each program consists of at least one
+   module, and every symbol is associated with some module.  If all the code of
+   a file is not enclosed in an explicit module, defined using the 'module'
+   keyword, then the file itself is treated as a module with the same name as
+   the file (minus the .chpl suffix).  The compiler can be directed to include
+   modules in separate files by naming them on the command line, utilizing the
+   -M flag, etc. (see the man page for exact details).
+ */
+module modToUse {
+
+  /* In this case, foo is a public module-level variable that is defined within
+     the module modToUse */
+  var foo = 12;
+
+  /* As is bar. */
+  var bar: int = 2;
+
+  /* A symbol can be declared "private" - this means that only code defined
+     within the same scope as the definition of this symbol can access it.
```

Comment:
I think that's on the right track.  How about simply "(including nested scopes)" or "(including code in nested scopes)"?  I think that'd be sufficiently clear (and "within those bounds" gives me pause).